assets/sass/v2/_select-authenticator.scss

```diff
@@ -100,6 +100,23 @@
     background-size: 38px 38px;
   }
 
+  .custom-auth-icon {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    width: 38px;
+    height: 38px;
+    border: 1px solid #ccc;
+    border-radius: 50%;
+    .custom-logo {
+      background-repeat: no-repeat;
+      background-position: 50% 50%;
+      background-size: cover;
+      width: 75%;
+      height: 75%;
```

Comment:
👋 Based on screenshots, it looks like something about the scaling implementation causes inconsistent outcomes between the list view and the challenge view? (The bird has less padding in the challenge view.)

Could we also stress test for forward compatibility of the SVG asset? The first-party graphics, sizing, spacing, background color, and customer theming of authenticator icons will change over time... ideally, an asset a customer provides today would be durable across those changes. I _think_ it does make sense for the widget to do padding or scaling somehow, and not have the customer bake padding into their asset. 🤔 But not sure, curious if @lesterchoi-okta has any thoughts for future proofing?

Reply:
The bird has less padding in challenge view because we had to extend the SVG icon end to end without adding too much padding programatically. for example, if admin needs to upload colored background SVG something like below, they would not see too much padding around.
<img width="567" alt="Screen Shot 2022-03-21 at 3 43 46 PM" src="https://user-images.githubusercontent.com/94395085/159353198-b5808bb8-d6cf-4d45-b510-c13450e3ea62.png">
 
The bird SVG we are using in this example is a draft version. May be it needs more padding in itself. We do have preview feature in admin screen, where they can upload different kind of logos to see if it fits perfectly for our usage. 

We also had a discussion regarding this in [slack](https://okta.slack.com/archives/C01DR8G2QBE/p1647892844770659?thread_ts=1647886715.796429&cid=C01DR8G2QBE)

Reply:
For now we are NOT going to add any padding from SIW Widget as per our UX suggestion. As this is Incremental development, we are happy to update it if UX feels otherwise. 
cc. @mauriciocastillosilva-okta